WeakAuras/Transmission.lua

```diff
@@ -726,16 +729,20 @@ local function recurseStringify(data, level, lines)
   end
 end
 
-function WeakAuras.DisplayToTableString(id)
+function WeakAuras.DisplayAuraToString(id)
```

Comment:
```suggestion
function WeakAuras.DataToString(id)
```
either that, or aura data, imo. If you don't care that much you can skip this comment.